cmd/earthly/main.go

```diff
@@ -400,15 +400,22 @@ func (app *earthlyApp) before(cliCtx *cli.Context) error {
 	if app.buildID == "" {
 		app.buildID = uuid.NewString()
 	}
-	if app.logstreamUpload {
-		app.logstream = true
-	}
-	disableOngoingUpdates := !app.logstream // TODO (vladaionescu)
+	disableOngoingUpdates := !app.logstream || app.interactiveDebugging
+	_, forceColor := os.LookupEnv("FORCE_COLOR")
+	_, noColor := os.LookupEnv("NO_COLOR")
 	var err error
-	app.logbusSetup, err = logbussetup.New(cliCtx.Context, app.logbus, app.debug, app.verbose, disableOngoingUpdates, app.logstreamDebugFile, app.buildID)
+	app.logbusSetup, err = logbussetup.New(
+		cliCtx.Context, app.logbus, app.debug, app.verbose, forceColor, noColor,
+		disableOngoingUpdates, app.logstreamDebugFile, app.buildID)
 	if err != nil {
 		return errors.Wrap(err, "logbus setup")
 	}
+	if app.logstreamUpload {
+		app.logstream = true
+	}
+	if app.logstream {
+		app.console = app.console.WithPrefixWriter(app.logbus.Run().Generic())
+	}
```

Comment:
nit:
```suggestion
	if app.logstreamUpload {
		app.logstream = true
		app.console = app.console.WithPrefixWriter(app.logbus.Run().Generic())
	}
```

Reply:
This is incorrect because `app.logstream` can be enabled separately too (not just via `app.logstreamUpload`).